### DISCUSSION TITLE
Automate data importation: extract data and import it directly

### DISCUSSION THREAD

**Discussion by JessicaRigol** [netbox-community/netbox#16743] [Help Wanted!]
I would like a solution to extract data and import it directly into Netbox, eliminating the need for the user to manually fill in all the connection and device information. 
Could this be done through Zabbix or another tool? 
I'm not sure if I'm being clear, as I am not yet proficient with automation tools. 

I appreciate any help!

Thanks!

**Comment by candlerb** [comment]
People have written various tools, but what you are asking for is strongly discouraged by the authors/designers of Netbox.

https://netboxlabs.com/docs/netbox/en/stable/introduction/#serve-as-a-source-of-truth
